lib/clients/rtm/client.js

```diff
@@ -704,6 +704,16 @@ RTMClient.prototype.updateMessage = function updateMessage(message, optCb) {
   return this._chat.update(message.ts, message.channel, message.text, message.opts, optCb);
 };
 
+/**
+ * Helper for deleting a sent message via the 'chat.delete' API call
+ * @param message {object} message The message object to be deleted,
+ * see /lib/clients/web/facets/chat.js for more details
+ * @param {Function=} optCb Optional callback
+ */
+RTMClient.prototype.deleteMessage = function deleteMessage(message, optCb) {
+  return this._chat.delete(message.ts, message.channel, message.text, message.opts, optCb);
```

Comment:
you don't pass text into this method. also, you haven't exposed the optional params like `as_user`.